daemon/cmd/daemon_main.go

```diff
@@ -663,6 +663,9 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.Bool(option.EnableIPv4Masquerade, true, "Masquerade IPv4 traffic from endpoints leaving the host")
 	option.BindEnv(vp, option.EnableIPv4Masquerade)
 
+	flags.Bool(option.EnableRemoteNodeSnat, false, "SNAT IPv4 packets from endpoints leaving the host destined to a remote node node in BPF routing mode. This option requires to set enable-bpf-masquerade to true.")
```

Comment:
```suggestion
	flags.Bool(option.EnableRemoteNodeMasquerade, false, "Masquerade packets from endpoints leaving the host destined to a remote node in BPF routing mode. This option requires to set enable-bpf-masquerade to true.")
```

Could you explain what you mean by "BPF routing mode" ? Which config option does that correspond to?

Reply:
Note that instead of hardcoding the `enable-bpf-masquerade` name, you can do a `fmt.Sprintf()` and refer to `option.EnableBPFMasquerade`. See other examples in the file.